vaxrank/cli.py

```diff
@@ -98,34 +125,28 @@
     type=int,
     help="Number of mutations to report")
 
-def main(args_list=None):
-    """
-    Script to generate vaccine peptide predictions from somatic cancer variants,
-    patient HLA type, and tumor RNA-seq data.
 
-    Example usage:
-        vaxrank
-            --vcf somatic.vcf \
-            --bam rnaseq.bam \
-            --vaccine-peptide-length 25 \
-            --output-csv vaccine-peptides.csv
-    """
-    if args_list is None:
-        args_list = sys.argv[1:]
-
-
-    args = arg_parser.parse_args(args_list)
-    logger.info(args)
-
-    if (len(args.output_csv) == 0 and
-            len(args.output_ascii_report) == 0 and
-            len(args.output_html_report) == 0 and
-            len(args.output_pdf_report) == 0):
-        raise ValueError(
-            "Must specify at least one of: --output-csv, "
-            "--output-ascii-report, "
-            "--output-html-report, "
-            "--output-pdf-report")
+def load_template_data(args):
+    if args.input_pickle_file:
+        f = open(args.input_pickle_file, 'rb')
+        template_data = pickle.load(f)
```

Comment:
Random suggestion for later code: http://preshing.com/20110920/the-python-with-statement-by-example/